core/block_svg.js

```diff
@@ -592,6 +595,63 @@ Blockly.BlockSvg.prototype.clearTransformAttributes_ = function() {
 };
 
 /**
+ * Transforms a block by setting the translation on the transform attribute
```

Comment:
Something's wrong here.  These should not be completely new functions in scratch-blocks.